sdk/eventgrid/azure-messaging-eventgrid/README.md

```diff
@@ -81,7 +81,7 @@ add the direct dependency to your project as follows.
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-messaging-eventgrid</artifactId>
-    <version>4.18.0</version>
+    <version>4.18.1</version>
```

Comment:
This should be a minor version update since the new version adds a bunch of new APIs for new events - `4.19.0`.

The build failure you see is because revapi is flagging that patch version update shouldn't have new API changes.